special_issues/2023_SI_PROMISE.md

```diff
@@ -0,0 +1,45 @@
+# Call for Papers ― EMSE Special Issue on “Trends in Predictive Models and Data Analytics in Software Engineering”
+
+A special issue of the Empirical Software Engineering Journal. 
+http://www.springer.com/computer/swe/journal/10664
+
+
+## Editors of the Special Issue
+* Steffen Herbold, University of Passau
+* Eunjong Choi, Kyoto Institute of Technology
+
+
+## Contact person
+* Steffen Herbold, University of Passau
```

Comment:
Would be good if you can indicate an email address to contact here. If you don't want to explicitly list your official email please "hide" it usign "(at)" or some similar way of obfuscating emails. Alternatively, create a specific, temp email address to use for the special issue. Thanks.